packages/rich-text-links/src/index.ts

```diff
@@ -29,13 +29,13 @@ function getRichTextEntityLinks(document: Document): EntityLinks {
   };
 }
 
-function addLinksFromRichTextNode(node: Node, links: EntityLinkMaps): void {
+function addLinksFromRichTextNode(node: Node, links: EntityLinkMaps, type?: string): void {
   const toCrawl: Node[] = [node];
 
   while (toCrawl.length > 0) {
-    const { data, content } = toCrawl.pop() as Block;
+    const { data, content, nodeType } = toCrawl.pop() as Block;
 
-    if (isLinkObject(data)) {
+    if ((!type || nodeType === type) && isLinkObject(data)) {
```

Comment:
please extract variable to clarify the purpose of this check.

Reply:
done